woocommerce/woocommerce-gutenberg-products-block
# Add product query support for Product Summary block

Adds Product Query support for the atomic Product Summary block

On the client side, when the Product Summary block is used within the product query block, the markup will be rendered on the server side - No javascript related to Product Summary block will be rendered.

Fixes #7334


### Testing
1. Checkout this branch and run npm start
2. Be sure that you have the latest version of the Gutenberg feature plugin installed and activated
3. Create a post/page and add the Product Query block.
4. Add Product Summary block to Post Template of product query block as shown in video below:

https://user-images.githubusercontent.com/16707866/204458088-0e719130-d858-4573-88b5-99d4922513ab.mov

5. Now save the post & confirm that: 
    1. No JS related to the Product Summary block is loaded (search in your network tab for product-summary-frontend.js). As you can see in the screenshot below, product-summary-frontend.js is showing. **This shouldn't be visible in the network tab.**
![image](https://user-images.githubusercontent.com/16707866/204456560-400521b1-3d17-479d-bce9-10f89f3cd9cb.png)
    2. UI of block should be identical in Frontend & Editor.
    3. The settings are reflected correctly (eg: custom style). Try changing custom style from the sidebar.
    4. ADDITIONAL CSS CLASS(ES)(available in advanced toggle in sidebar) should be added to the container div
    5. Summary text styling set using WYSIWYG editor should work in Editor & frontend. By WYSIWYG editor, I mean following editing block of product:
![image](https://user-images.githubusercontent.com/16707866/204457376-3313ca80-9b91-47d6-9802-cafbd8c306ca.png)





### WooCommerce Visibility
